vite.config.js

```diff
@@ -12,12 +13,27 @@ export default defineConfig({
   plugins: [
     react(),
     eslintPlugin({
-      cache: false, // disable eslint cache to avoid conflicts
+      cache: false,
     }),
+    nodePolyfills(),
   ],
   build: {
     sourcemap: true,
     emptyOutDir: true,
+    rollupOptions: {
+      onwarn: (warning, defaultHandler) => {
+        // TODO: remove isFromKbssCvutPackageWarning and solve the root cause, see https://github.com/kbss-cvut/record-manager-ui/issues/113
```

Comment:
This is not very informative. I do not understand why you are filtering this. Please write the reason here if it is  ok to filter or try to find solution without filtering the warning.